cms/envs/aws.py

```diff
@@ -210,6 +211,18 @@
 TENDER_DOMAIN = ENV_TOKENS.get('TENDER_DOMAIN', TENDER_DOMAIN)
 TENDER_SUBDOMAIN = ENV_TOKENS.get('TENDER_SUBDOMAIN', TENDER_SUBDOMAIN)
 
+# Modules having these categories would be excluded from progress calculations
```

Comment:
Nit: Please include more information here about the purpose of these fields and why they are different from what is defined in lms/envs/aws.py